cpp/ps0005/main.cpp

```diff
@@ -4,47 +4,65 @@ using namespace std;
 
 int main()
 {
-    int a,b=0;
-    float c,d=0;
+    // prompting user for 2 operands and an operator 1.5/2
+    float num1, num2;
     char proc;
+    
     cout<<"please Enter Two number :)\nnow you can Enter frist number a= ";
-    cin>>a;
+    cin>>num1;
     if(cin.fail()==true)
     {
-    cout<<"Error please input number\n";
-    system("pause");
-    return 0;
+        cout<<"Error please input number\n";
+        system("pause");
+        return 5;
     }
-    else
+    
     cout<<"now you can Enter scond number b= ";
-    cin>>b;
+    cin>>num2;
     if(cin.fail()==true)
     {
-    cout<<"Error please input number\n";
-
-    system("pause");
-    return 0;
+        cout<<"Error please input number\n";
+        system("pause");
+        return 5;
     }
-    else
+    
+    int a=num1, b=num2;
+    
     cout<<"Choose operation (+,-,*,/): \n";
     cin>>proc;
+

```

Comment:
- [x] declared variables to store the user input
- [ ] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input